packages/fiori/test/pages/styles/DynamicSideContent.css

```diff
@@ -0,0 +1,26 @@
+body, html {
+	width: 100%;
+	height: 100%;
+	padding: 0;
+	margin: 0;
+}
+
+.page1auto {
+	background-color: var(--sapBackgroundColor);
+}
+
+.content-padding * {
+	padding: 0.5rem;
+}
+
+:root {
+	--_ui5_input_width: 10rem !important;
```

Comment:
Are you sure that this is needed? Can't you set the width for the ```ui5-input``` tag

Reply:
Fixed with another approach.